barretenberg/cpp/src/barretenberg/vm2/simulation/nullifier_tree_check.test.cpp

```diff
@@ -43,9 +44,10 @@ TEST(AvmSimulationNullifierTree, ReadExists)
     EXPECT_CALL(merkle_check, assert_membership(low_leaf_hash, low_leaf_index, _, snapshot.root))
         .WillRepeatedly(Return());
```

Comment:
Do you need `.WillRepeatedly(Return());` ? (I know you didn't change it in this file)

I think you can just expect a call.

Reply:
Ah got it. I thought the default `EXPECT_CALL` would let you call it any number of times but apparently not: https://google.github.io/googletest/reference/mocking.html#EXPECT_CALL.Times